test/SlackIntegration.test.ts

```diff
@@ -1,24 +1,33 @@
-import * as SlackInteraction from './slackInteraction.json';
-import { parsePayloadFromEvent } from '../src/SlackInteractivityLambda/SlackInteractivity.lambda';
+import * as crypto from 'crypto';
+import { authenticate } from '../src/SlackInteractivityLambda/SlackInteractivity.lambda';
 
-function buildBase64Payload() {
-  const a = JSON.stringify(SlackInteraction);
-  const b = encodeURIComponent(a);
-  const c = `payload=${b}`;
-  const d = Buffer.from(c).toString('base64');
-  return d;
-}
+process.env.SLACK_SECRET_ARN = 'arn:blabla';
+process.env.AWS_REGION = 'eu-central-1';
 
-test('decode payload', () => {
+test('No tests yet', () => {
 
-  // Well... event is a large object, circumventing the typechecker 😬
+  const timestamp = Math.floor(Date.now() / 1000);
+  console.log(timestamp);
+  const body = 'payload=%7Btest%7D';
+  const slackSecret = 'OAEIJegjwogj0239230598weogijf';
```

Comment:
Is dit een echt secret?

Reply:
Nee dit is een random test string. Slack signing secret wordt opgehaald uit de secrets manager via de lambda.